### PR TITLE
[fix](load) Fix load channel mgr lock

### PR DESCRIPTION
**Pull request by xinyiZzz**
# Proposed changes

Issue Number: close #xxx

## Problem summary

Describe your changes.

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [x] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [x] No
    - [ ] No Need
3. Has document been added or modified:
    - [ ] Yes
    - [x] No
    - [ ] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [x] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [x] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by morningman — DISMISSED** [review]
LGTM

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 38.03 seconds
 load time: 605 seconds
 storage size: 17182969197 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20221104004153_clickbench_pr_39631.html

File: be/src/runtime/load_channel_mgr.h [hunk 1]
```diff
@@ -85,6 +88,7 @@ class LoadChannelMgr {
     std::mutex _lock;
     // load id -> load channel
     std::unordered_map<UniqueId, std::shared_ptr<LoadChannel>> _load_channels;
+    SpinLock _load_channels_lock;
     std::shared_ptr<LoadChannel> _reduce_memory_channel = nullptr;
```
Comment (yiguolei) [inline be/src/runtime/load_channel_mgr.h:92]: This lock's scope is same as _lock, I think we could just use _lock.

**Review by yiguolei — DISMISSED** [review]
LGTM

File: be/src/runtime/load_channel_mgr.h (hunk 1 above)
Comment (k-i-d-d) [reply be/src/runtime/load_channel_mgr.h]: done

Comment (xinyiZzz) [reply be/src/runtime/load_channel_mgr.h]: done

**Review by dataroaring — APPROVED** [review]
LGTM

**Review by morningman — APPROVED** [review]

**Pull request merged**